hub/instruction_handling/create/_local.tex

```diff
@@ -38,6 +39,8 @@
 \def\locMxpMtntop                                   {\col{\mxpMod\_mtntop}}
 %
 \def\locTriggerOob                                  {\col{trigger\_\oobMod}}
+\def\locTriggerOobX                                 {\col{trigger\_\oobMod\_X}}
+\def\locTriggerOobU                                 {\col{trigger\_\oobMod\_U}}
```

Comment:
Two different OOB module triggers to keep things clean.

**U** stands for _unexeptional_
**X** stands for _exceptional_